lib/parser.js

```diff
@@ -92,7 +93,7 @@ function parseCues (cues) {
 function parseCue (cue, i) {
   let identifier = '';
   let start = 0;
-  let end = 0;
+  let end = 0.01;
```

Comment:
Why is this?

Reply:
End time must be greater than start time.